src/devtools/client/debugger/src/actions/breakpoints/breakpoints.ts

```diff
@@ -317,14 +321,14 @@ export function addBreakpointAtColumn(cx: Context, location: Location): UIThunkA
   };
 }
 
-export function setBreakpointPrefixBadge(breakpoint: Breakpoint, prefixBadge: string) {
-  return {
-    breakpoint: {
-      ...breakpoint,
-      options: { ...breakpoint.options, prefixBadge },
-    },
-    type: "SET_BREAKPOINT",
-  };
+export function setBreakpointPrefixBadge(
+  breakpoint: Breakpoint,
+  prefixBadge: Breakpoint["options"]["prefixBadge"]
```

Comment:
Just curious — what's the benefit of what looks to be a computed (?) type here instead of just string? I'm guessing it's just so if we change prefixBadge this gets updated as well?

Reply:
The actual type is an enum/union of some kind, so it was faster to just point to "whatever the type of that field is" than look it up :)